resources/istio/templates/rbac-migration-remove-dr.yaml

```diff
@@ -0,0 +1,39 @@
+---
+apiVersion: v1
+kind: ServiceAccount
```

Comment:
Do we need that? this DR will be removed on upgrade by tiller?

Reply:
The DR is not managed by tiller, we create by with kubectl in the install job, so we need to remove it during upgrades

Reply:
Yeah, I forgot that this was changed and is applied as a part of istio-installer job :)